pkg/test/istio.io/tasks/security/authn-policy/wait-for-containers.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+# Wait for all containers to be ready, for up to 2 minutes = 12 * 10 seconds.
+retries=12
+while [ "$retries" -gt "0" ]; do
+  if kubectl get pods --all-namespaces -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | grep -q -v false; then
```

Comment:
(doesn't need to be in this PR) the test framework is going to need some retrying strategy, otherwise every script is going to need this kind of logic

Reply:
Yes, agreed. There are implicit delays and retries that are not explicit in the docs. There will be tons of those.